rllib/offline/estimators/direct_method.py

```diff
@@ -1,80 +1,22 @@
 import logging
-from typing import Tuple, Generator, List
-from ray.rllib.offline.estimators.off_policy_estimator import (
-    OffPolicyEstimator,
-    OffPolicyEstimate,
-)
+from typing import Dict, Any
+from ray.rllib.offline.estimators.off_policy_estimator import OffPolicyEstimator
 from ray.rllib.policy import Policy
-from ray.rllib.policy.sample_batch import SampleBatch
+from ray.rllib.policy.sample_batch import MultiAgentBatch, DEFAULT_POLICY_ID
 from ray.rllib.utils.annotations import DeveloperAPI, override
 from ray.rllib.utils.framework import try_import_torch
-from ray.rllib.utils.numpy import convert_to_numpy
 from ray.rllib.utils.typing import SampleBatchType
-from ray.rllib.offline.estimators.fqe_torch_model import FQETorchModel
-from ray.rllib.offline.estimators.qreg_torch_model import QRegTorchModel
-from gym.spaces import Discrete
+from ray.rllib.utils.numpy import convert_to_numpy
 import numpy as np
 
 torch, nn = try_import_torch()
 
 logger = logging.getLogger()
 
 
-@DeveloperAPI
-def train_test_split(
```

Comment:
users have to create train test split on their own I'm assuming

Reply:
by passing a separate file to the eval config

Reply:
Yup